opennms-doc/guide-development/src/asciidoc/text/rest/measurements.adoc

```diff
@@ -16,8 +16,9 @@ The following table shows all supported query string parameters and their defaul
 [options="header"]
 |===
 | name               | default   | comment
-| start              | -14400000 | Timestamp in milliseconds. If < 0, the effective value will be (end + start).
-| end                | 0         | Timestamp in milliseconds. If <= 0, the effective value will be the current timestamp.
+| start              | -14400000 | Timestamp in milliseconds. If < 0, the effective value will be `end - start`
```

Comment:
The other updates you made are fine, but `end - start` is not accurate.

Let's assume that end=0 and start=-14400000 (the defaults), and now=1468273176000.

The effective values are:
end=now=1468273176000
start=end + start=1468273176000 + (-14400000)=1468258776000